src/core/src/tortuga/scripts/get_software_profile.py

```diff
@@ -131,6 +135,9 @@ def runCommand(self):
         if self.getArgs().getDataRoot:
             optionDict['dataRoot'] = True
 
+        if self.getArgs().getDataRoot:
```

Comment:
Is this line correct here? Shouldn't it be `if self.getArgs().getDataRsync:`?

Reply:
right... fixed variable name